modules/apps/feature-flag/feature-flag-test/src/testFunctional/tests/FeatureFlag.testcase

```diff
@@ -111,6 +111,16 @@ definition {
 		}
 
 		task ("Then a feature flag that depends on the enabled feature flag can be enabled") {
+			WaitForElementPresent(
+				locator1 = "FeatureFlag#TOGGLE_ROW",
+				title_row = "Dependency FF Test",
+				toggle_state = "Disabled");
+
+			ScrollWebElementIntoView(
+				locator1 = "FeatureFlag#TOGGLE_ROW",
+				title_row = "Dependency FF Test",
+				toggle_state = "Disabled");
+
```

Comment:
Is this necessary? Remnants from troubleshooting?

Reply:
I have the impression you told me to add those lines to mimic the user's behavior. I'll remove them now!

Reply:
Done